operations/restrictions.md

```diff
@@ -189,6 +189,14 @@ Returns all restrictions of the default service provided by the enterprise.
 
 Adds new restrictions with the specified conditions.
 
+**Important:** If consecutive restrictions are sent with the exact same conditions and exceptions, no attempt at merging them into a single restriction is made. This means that there can be a large number of restrictions per service, leading to sub-optimal performance. A quota limit of 150000 has been introduced for this reason. To mitigate the issue, the preferred way to add restrictions is new operation [set restrictions](#set-restrictions). The new operation is currently marked as 'Restricted' and subject to change as part of beta testing, but in time we expect that to replace [add restrictions](#add-restrictions).
+
+This means that there can easily be a big number of restrictions per service, leading to sub-optimal performance. Quota of **150000** has been introduced for this reason.
```

Comment:
These three paragraphs can be removed, as they are replaced by the paragraph above in the block quotes.